### ISSUE TITLE
JAWS jumps to top of page when clicking an element without focus

### ISSUE THREAD

**Issue by danielghost** [adaptlearning/adapt-contrib-core#696]
When using JAWS, clicking on a element which doesn't receive focus, after one which does, results in the page jumping back to the top of the page.

With `_fixes._safariLabelClickBlur` enabled in _config.json_, this also occurs when clicking an element that doesn't return early from the following (e.g. `mcq-item__input`) as the focus isn't applied to the clicked item:
https://github.com/adaptlearning/adapt-contrib-core/blob/c7c74d45b3cf0751455658538a585d41db0bf340/js/fixes/safari.label.click.blur.js#L34
